index.html

```diff
@@ -457,20 +465,20 @@ <h4>DataIntegrityProof</h4>
           </p>
           <p>
 The `cryptosuite` property of the proof MUST be
-`experimental-quantum-safe-dilithium-2024`.
+`experimental-ml-dsa-2024`.
```

Comment:
we used "experimental-mldsa44-2024" as ml-dsa-44 is the only algo we implemented so far